app/serializers/discussion_reader_serializer.rb

```diff
@@ -0,0 +1,18 @@
+class DiscussionReaderSerializer < ActiveModel::Serializer
+  embed :ids, include: true
+
+  attributes :id,
+             :discussion_reader_id,
+             :read_items_count,
+             :read_salient_items_count,
+             :last_read_sequence_id,
+             :last_read_at
+
+  def id
+    object.discussion_id
```

Comment:
Here the 'id' is actually the discussion's id, to ensure that the clientside model is updated correctly.